client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -148,7 +148,7 @@ class Dashboard extends Component {
 		}
 
 		if ( ! setStoreAddressDuringInitialSetup && ! hasProducts ) {
-			return <PreSetupView site={ selectedSite } />;
+			return <PreSetupView siteId={ selectedSite.ID || null } />;
```

Comment:
It appears elsewhere in this file we always check for the presence of `selectedSite` - should we do the same here to avoid a possible situation where it might be undefined?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/19413/commits/48914387c2e937112afeb8eebee34f32a16b6ef2